tests/test_commands.py

```diff
@@ -2382,6 +2382,37 @@ def test_geoadd(self, r):
         assert r.geoadd('barcelona', *values) == 2
         assert r.zcard('barcelona') == 2
 
+    @skip_if_server_version_lt('6.2.0')
+    def test_geoadd_nx(self, r):
+        values = (2.1909389952632, 41.433791470673, 'place1') + \
```

Comment:
This API usage doesn't look anything like the samples on the redis commands page. Can you see what you can do to align the API to be more like that one - and then update the tests?